mlflow/models/evaluation/default_evaluator.py

```diff
@@ -0,0 +1,651 @@
+        legend_loc = "lower right"
+    elif curve_type == "pr":
+
+        def gen_x_y_thresholds_fn(_y, _y_prob):
+            precision, recall, _thresholds = sk_metrics.precision_recall_curve(_y, _y_prob)
+            _thresholds = np.append(_thresholds, [1.0], axis=0)
+            return recall, precision, _thresholds
+
+        xlabel = "recall"
+        ylabel = "precision"
+        legend_loc = "lower left"
+    else:
+        assert False, "illegal curve type"
+
+    if is_binomial:
+        x_data, y_data, thresholds = gen_x_y_thresholds_fn(y, y_probs)
+        data_series = [("positive class", x_data, y_data)]
+        legend_loc = None
+        auc = sk_metrics.auc(x_data, y_data)
+    else:
+        curve_list = []
+        for positive_class_index, positive_class in enumerate(labels):
+            y_bin, _, y_prob_bin = _get_binary_sum_up_label_pred_prob(
+                positive_class_index, positive_class, y, None, y_probs
+            )
+
+            x_data, y_data, thresholds = gen_x_y_thresholds_fn(y_bin, y_prob_bin)
+            curve_list.append((positive_class, x_data, y_data, thresholds))
+
+        data_series = [
+            (f"Positive Class = {positive_class}", x_data, y_data)
+            for positive_class, x_data, y_data, _ in curve_list
+        ]
+        auc = [sk_metrics.auc(x_data, y_data) for _, x_data, y_data, _ in curve_list]
+    return _Curve(
+        plot_fn=plot_lines,
+        plot_fn_args={
+            "data_series": data_series,
+            "xlabel": xlabel,
+            "ylabel": ylabel,
+            "legend_loc": legend_loc,
+            "line_kwargs": {"drawstyle": "steps-post"},
+        },
+        auc=auc,
+    )
+
+
+_matplotlib_config = {
+    "figure.dpi": 144,
+    "figure.figsize": [6.0, 4.0],
+}
+
+
+# pylint: disable=attribute-defined-outside-init
+class DefaultEvaluator(ModelEvaluator):
+    def can_evaluate(self, model_type, evaluator_config=None, **kwargs):
+        return model_type in ["classifier", "regressor"]
+
+    def _log_metrics(self):
+        """
+        Helper method to log metrics into specified run.
+        """
+        timestamp = int(time.time() * 1000)
+        self.client.log_batch(
+            self.run_id,
+            metrics=[
+                Metric(
+                    key=_gen_log_key(key, self.dataset_name),
+                    value=value,
+                    timestamp=timestamp,
+                    step=0,
+                )
+                for key, value in self.metrics.items()
+            ],
+        )
+
+    def _log_image_artifact(
+        self,
+        do_plot,
+        artifact_name,
+    ):
+        import matplotlib.pyplot as pyplot
+
+        artifact_file_name = _gen_log_key(artifact_name, self.dataset_name) + ".png"
+        artifact_file_local_path = self.temp_dir.path(artifact_file_name)
+
+        try:
+            pyplot.clf()
+            do_plot()
+            pyplot.savefig(artifact_file_local_path)
```

Comment:
Can we set the dpi value to something higher than the default of 100? The images look really blurry and out of place in the tracking UI. 300 might be a better option.
```python
pyplot.savefig(artifact_file_local_path, dpi=300)
```

Reply:
Or perhaps save in .svg format to ensure that scalability in different zoom levels in browsers will work well?

Reply:
I increase dpi to 288 (previous in databricks notebook it is 72)

Save fig as svg is not good choice, the Pillow package does not support svg. But we need return artifact as the pillow Image instance.